.gitignore

```diff
@@ -110,6 +110,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+list/
```

Comment:
What is the `list/` folder?

Reply:
Was created as part of a virtual environment. Not sure how that got there, but it's been removed.